CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 - [BREAKING] Add error messages to errors and implement `core::error::Error` (#974).
 - Implemented new `digest_from_hex!` macro (#984).
 - Added Format Guidebook to the `miden-lib` crate (#987).
+- Added `miden::note::get_script_hash` procedure (#995).
```

Comment:
I think we should note this change as breaking, since it changes the offsets of the kernel procedures.